src/sqlfluff/dialects/dialect_snowflake_keywords.py

```diff
@@ -594,4 +595,7 @@
 XML
 ZONE
 ZSTD
+'AZURE'
+'GCS'
+'S3'
```

Comment:
See my note on `dialect_snowflake.py` - I think we should remove these quoted variants.

Reply:
Should now be resolved.